### ISSUE TITLE
About the training process

### ISSUE THREAD

**Issue by dylansheng** [a-mitm/CommonsenseQAWithBERT#1]
Sorry to bother you. When fine-tuning the model using GPU, I face that "It seems that global step (tf.train.get_global_step) has not been increased. Current value (could be stable): 10 vs previous value: 10. You could increase the global step by passing tf.train.get_global_step() to Optimizer.apply_gradients or Optimizer.minimize".  Does this mean that the model can finish the training process? But I found that the accuracy of the dev dataset is just 30%, which is far from the standard, which is 52% in the leaderboard. https://www.tau-nlp.org/csqa-leaderboard2. Looking forward to your reply. Thanks!

**Comment by dylansheng** [comment]
Sorry I have misuderstood this term~
